src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -189,12 +196,30 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
 
     file_stat->size = (handle->ffd.nFileSizeHigh * ((int64_t)(MAXDWORD)+1)) + handle->ffd.nFileSizeLow;
     file_stat->modtime = FileTimeToUnixTime(&handle->ffd.ftLastWriteTime, &rem);
+
+    std::wstring fullPath;
+    fullPath.reserve(std::wcslen(handle->path) + std::wcslen(handle->ffd.cFileName));
+    fullPath += handle->path; // path always ends with '/', by construction
```

Comment:
That would be a `\` and not a `/`.

Reply:
I was wondering the same.

Line 65 do `strncat(h, "/*", 2);`
So it is a `/`.   I tested and it works like this.
Should I change it to a `\`  when i remove the '*' earlier ?

Reply:
From IRC discussion: No I just mean the comment, dirname went through c_utf8_path_to_locale, where / is converted to \\. So the comment lower should say // path always ends with '\\', by construction